src/app/pages/backend-integration/integration-description/integration-description.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, Input} from '@angular/core';
+
+@Component({
+  selector: 'ngx-integration-description[features][url][buttonText]',
```

Comment:
Why do we need attribute selectors here? Is it some kind of required inputs hack? If so, I believe it's better to check if inputs set in some lifecycle hook and throw an error with an explanation of the error. Otherwise, the component wouldn't render or there will be an generic Angular error about unknown element.